astro.config.mjs

```diff
@@ -40,6 +40,15 @@ export default defineConfig({
         dark: "./src/assets/logo-dark.png",
       },
       favicon: "./src/assets/logo.png",
+      head:[
+        {
+          tag:"meta",
+          attrs:{
+            property:"og:image",
+            content:"./src/assets/ratatui-og.png"
```

Comment:
```suggestion
            content: "https://private-user-images.githubusercontent.com/1813121/290281077-352abbe5-0032-44ba-aae7-0e5b3d5dff4b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0ODk3NTIsIm5iZiI6MTcwMjQ4OTQ1MiwicGF0aCI6Ii8xODEzMTIxLzI5MDI4MTA3Ny0zNTJhYmJlNS0wMDMyLTQ0YmEtYWFlNy0wZTViM2Q1ZGZmNGIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMTNUMTc0NDEyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NWQ5YjExYzQwNzJiY2ZiNjYxOWI5ZmE4ZThjZTc5NDQ3NWI4ZjY0YmU1YjExNzg1NTY0NGJlNGU5OWE5ODJlMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.dvS3bJtJP2cnCXERlkRj3oY9nvn9UC7Ae6FIwbcAMFk"
```

Reply:
I'd prefer to have something that's more easily editable than a random github image link like this.

Reply:
The problem is astro adds this as the link in the meta: 

`https://ratatui.rs/./src/assets/ratatui-og.png`